### PR TITLE
Update en.json

### PR DESCRIPTION
**Pull request by priiduonu**
fix typo

### CODE REVIEW

**Comment by Thomas55555** [comment]
thx @priiduonu

File: custom_components/husqvarna_automower/translations/et.json [hunk 1]
```diff
@@ -43,33 +43,33 @@
...
-					"park_override": "Parkimise tühistamine",
-					"sensor": "Ilmataimer",
-					"daily_limit": "Päevane limiit",
-					"fota": "Püsivara uuendamine",
-					"frost": "Härmatis"
-				}
-			},
-			"mode_list":{
-				"state": {
+    "entity": {
```
Comment (Thomas55555) [inline custom_components/husqvarna_automower/translations/et.json:46]: Is there a reason, why you change des number of spaces?
I want to keep the translations and the strings.json all in the same format

Comment (priiduonu) [reply custom_components/husqvarna_automower/translations/et.json:46]: I noted that some indenting was done with tabs, some with spaces. I did not wat to change the number of spaces, it was probably done by github editor. I'm a bit confused, please disregard this PR alltogether.

Comment (Thomas55555) [reply custom_components/husqvarna_automower/translations/et.json:46]: Ok

**Pull request closed without merging**